### PR TITLE
Add openLDAP support

### PR DESCRIPTION
**Pull request by sanjay990**
Added support for running product-tests on openLDAP docker container.
This will enable us to run ldap product-tests on Travis.

### CODE REVIEW

**Comment by sanjay990** [comment]
@ArturGajowy @amrutagokhale @anusudarsan @arhimondr

File: presto-product-tests/conf/presto/etc/singlenode-ldap.properties [hunk 1]
```diff
@@ -17,13 +17,14 @@ discovery-server.enabled=true
...
 authentication.ldap.enabled=true
-authentication.ldap.url=ldaps://ldap-server:636
-authentication.ldap.ad-domain=presto.testldap.com
-authentication.ldap.base-dn=OU=Asia,DC=presto,DC=testldap,DC=com
-authentication.ldap.group-dn=CN=ParentGroup,OU=America,DC=presto,DC=testldap,DC=com
+authentication.ldap.server-type=OPENLDAP
+authentication.ldap.user-object-class=inetOrgPerson
+authentication.ldap.url=ldaps://ldapserver:636
+authentication.ldap.base-dn=ou=Asia,dc=presto,dc=testldap,dc=com
+authentication.ldap.group-dn=cn=DefaultGroup,ou=America,dc=presto,dc=testldap,dc=com
```
Comment (anusudarsan) [inline presto-product-tests/conf/presto/etc/singlenode-ldap.properties:30]: I see you have created the DefaultGroup in openLDAP. Did you also create it in AD?

**Comment by anusudarsan** [comment]
LGTM % question

File: presto-product-tests/conf/presto/etc/singlenode-ldap.properties (hunk 1 above)
Comment (sanjay990) [inline presto-product-tests/conf/presto/etc/singlenode-ldap.properties:30]: @anusudarsan : No, this has to be created on AD.

**Comment by ArturGajowy** [comment]
Please create a branch based on this branch with the ldap product tests run on travis, so that we know it works on travis :)

Let's **not** have ldap product tests run on every presto build, we'll include them in nightly builds (on which I'm going start working soon)

**Comment by amrutagokhale** [comment]
LGTM.

**Pull request closed without merging**